docs/node-operators/get-started/setup-validator.md

```diff
@@ -143,9 +142,9 @@ Use a service manager (for example Systemd) to run Visor. While you can use any
 ```
 
 ## Modify Vega config
-When announcing your node (below), the node will need to be pointing to a Tendermint node. Set that in your Vega config based on information in the Tendermint config file. 
+When announcing your node (below), the node will need to be pointing to a CometBFT node. Set that in your Vega config based on information in the CometBFT config file. 
 
-Find the address for the RPC server in your Tendermint config `YOUR_TENDERMINT_HOME_PATH/config/config.toml`, under the heading `TCP or UNIX socket address for the RPC server to listen on`. 
+Find the address for the RPC server in your CometBFT config `YOUR_COMETBFT_HOME_PATH/config/config.toml`, under the heading `TCP or UNIX socket address for the RPC server to listen on`. 
```

Comment:
We have to add also something like the following in the `Point to Arbitrum node` (see the issue I have created here: https://github.com/vegaprotocol/vega/issues/11684) :


Add config for the Event Forwarded for the Arbitrum endpoint:

```toml title=YOUR_VEGA_HOME/config/node/config.toml
...
[EvtForward]
...


  [[EvtForward.EVMBridges]]
    Level = "Info"
    MaxEthereumBlocks = 500
    PollEventRetryDuration = "500ms"
    ChainID = "421614"
    SkipClientVerification = true
    HeartbeatIntervalForTestOnlyDoNotChange = "0s"
```